src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -66,14 +72,18 @@ public class LinkTagHelper : TagHelper
         private enum Mode
         {
             /// <summary>
+            /// Adding file version to the generated urls.
```

Comment:
my comment here was that you were changing more than "src" URLs, not that the other bit was misleading.